app/javascript/crayons/Links/__stories__/Links.mdx

```diff
@@ -27,10 +27,9 @@ It is recommended to always use `branded` variant for inline links, especially w
 
 ### Icons
 
-Icons are only supported in *block* links (`block`).
-Icon can be placed in front of a label to both clarify an action and call attention to a link.
-By passing an object in `src` prop, there will be automatically included `<Icon />` component.
-By default, the icon itself won't be customizable.
+Icon can be placed in front of a label to both clarify an action but also to make link stand out more.
```

Comment:
**_Quick suggestion_**: (same as my last two suggestions!) let's make "Icon" plural here and add an "a" before "link" (or remove the "a" and make "link" plural instead 😄 )!
```suggestion
Icons can be placed in front of a label to both clarify an action but also to make a link stand out more.
```